src/testFiles/missionSnapShots.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
- [ ] Kindly make sure to add tests for all components and functions as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=NOTE%3A%20Make%20sure%20to%20add%20test%20for%20all%20your%20components%20and%20functions.).

Reply:
@gemmen29 thanks for the review. We will do that.